float64 latitude     # 纬度 (degrees)
float64 longitude    # 经度 (degrees)
float64 altitude     # 高度 (m)
float64 velocity     # 速度 (m/s)
float64 heading      # 朝向 (rad)
float64 yaw_goal_tolerance  # 航向角容差（弧度）
float64 xy_goal_tolerance   # 平面位置容差（米） 